t/022-redirect.t

```diff
@@ -218,3 +218,54 @@ GET /read
 Location: http://agentzh.org/foo?a=b&c=d
 --- response_body_like: 307 Temporary Redirect
 --- error_code: 307
+
+
+
+=== TEST 12: explicit 303
+--- config
+    location /read {
+        content_by_lua '
```

Comment:
That would seem sensible from my point of view. Do you want me to change that here - so the tests are changed when they're touched - or do you want me to leave this and have that done separately?